cpmpy/solvers/minizinc.py

```diff
@@ -382,7 +382,7 @@ def transform(self, cpm_expr):
 
         :return: list of Expression
         """
-        return toplevel_list(cpm_expr)
+        return simplify_boolean(toplevel_list(cpm_expr))
```

Comment:
I added this because it fixes a bug where minizinc does not recognice 'False' when doing something like IV == False, but that can indeed be done with a small transformation as well